### PR TITLE
http to https

### PR DESCRIPTION
**Pull request by jburel**
Use https instead of http

cc @sbesson

### CODE REVIEW

**Comment by sbesson** [comment]
Overall looks good, tested by running `grep "http://www.openmicroscopy.org\|http://docs.openmicroscopy\|http://downloads.openmicroscopy\|writing output"` on the CI job log [without this PR](https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs/345/consoleText) and [with this PR](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/842/consoleText).

A couple of remaining hardcoded HTTP links seem to remain in the OMERO docs - see [http.txt](https://github.com/openmicroscopy/ome-documentation/files/1302728/http.txt) for the output of the grep with this PR included. Few comments:

- the `users/history.txt` HTTP links are addressed upstream in https://github.com/openmicroscopy/openmicroscopy/pull/5473
- the `cvs.openmicroscopy.org.uk` redirects should probably be rewritten to redirect to HTTPS
- I have not reviewed the contributing logs but I can do so if desired

**Comment by jburel** [comment]
Few remaining "http" than will need to be fixed in other repositories: ``users/history.txt`` (openmicroscopy) and the various walkthroughs (omero-install/omeroweb-install)

File: omero/developers/json-api.rst [hunk 1]
```diff
@@ -432,7 +432,7 @@ These query parameters are also supported (see above):
           "url:project": "http://server.openmicroscopy.org/api/v0/m/projects/11601/",
           "url:datasets": "http://server.openmicroscopy.org/api/v0/m/projects/11601/datasets/",
           "@id": 11601,
-          "@type": "http://www.openmicroscopy.org/Schemas/OME/2016-06#Project",
```
Comment (sbesson) [inline omero/developers/json-api.rst]: Just realized this is not the case in `omero-marshal` although we should probably consider using HTTPS in  https://github.com/openmicroscopy/omero-marshal/blob/master/omero_marshal/__init__.py#L63 /cc @chris-allan

Comment (jburel) [reply omero/developers/json-api.rst]: I have it on my todo list i.e. update omero-marshal

**Comment by jburel** [comment]
``https://cvs.openmicroscopy.org.uk/`` cannot be used at the moment
Users will be sent to a ``Your connection is not private`` landing page

**Comment by jburel** [comment]
Fix for walkthroughs: https://github.com/ome/omero-install/pull/166

**Comment by jburel** [comment]
omero-marshal: https://github.com/openmicroscopy/omero-marshal/pull/45

File: omero/developers/json-api.rst (hunk 1 above)
Comment (chris-allan) [reply omero/developers/json-api.rst]: We could certainly do this going forward but this breaks all the backwards compatibility. I would not change these.

**Review by chris-allan — CHANGES_REQUESTED** [review]

File: omero/developers/json-api.rst (hunk 1 above)
Comment (jburel) [reply omero/developers/json-api.rst]: I will roll the change back in the example

**Comment by jburel** [comment]
@chris-allan: Done

File: contributing/data-model-schema.rst [hunk 2]
```diff
@@ -331,7 +331,7 @@ Documentation for the released schema must be generated from the
 this task, and requires the schema definitions to have been published online
 as described above. To build the generated documentation for a given release::
 
-  /Applications/oxygen/schemaDocumentationMac.sh http://www.openmicroscopy.org/Schemas/OME/$RELEASE/ome.xsd -cfg:components/specification/omeOxygenDocConfig.xml
+  /Applications/oxygen/schemaDocumentationMac.sh https://www.openmicroscopy.org/Schemas/OME/$RELEASE/ome.xsd -cfg:components/specification/omeOxygenDocConfig.xml
```
Comment (mtbc) [inline contributing/data-model-schema.rst:334]: I'm assuming that whatever this does can cope with https.

File: omero/sysadmins/omeroweb-upgrade.rst [hunk 3]
```diff
@@ -70,6 +70,7 @@ reason, it is possible that an update of OMERO will cause issues with an older
 version of a plugin. It is best when updating the server to also install any
 available plugin updates according to their own documentation.
 
+
```
Comment (mtbc) [inline omero/sysadmins/omeroweb-upgrade.rst]: accidental?

File: contributing/data-model-schema.rst (hunk 2 above)
Comment (jburel) [reply contributing/data-model-schema.rst:334]: it does

**Comment by jburel** [comment]
Any more concerns/comments since this PR is blocking other PRs?

**Review by chris-allan — APPROVED** [review]

File: omero/developers/json-api.rst [hunk 4]
```diff
@@ -58,7 +58,7 @@ OME schema but is not identical. In the cases where OMERO-specific fields are in
 will be prefixed by ``omero:``. For example, ``omero:details`` specifies the owner, group and
 permissions of each object in OMERO.
 JSON objects also include an ``@id`` of the object in the OMERO database and a ``@type`` that specifies
-the OME Schema used to generate it such as ``http://www.openmicroscopy.org/Schemas/OME/2016-06#Project``.
+the OME Schema used to generate it such as "http://www.openmicroscopy.org/Schemas/OME/2016-06#Project".
```
Comment (sbesson) [inline omero/developers/json-api.rst]: This is not a valid anchor and currently breaks https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/845

**Comment by jburel** [comment]
@sbesson https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-merge-docs/846/ is back to green

**Review by sbesson — APPROVED** [review]

**Comment by jburel** [comment]
Thanks all
Merging

**Pull request merged**